src/screen.rs

```diff
@@ -7,6 +7,8 @@
 extern crate rustyline;
 #[cfg(feature = "pretty")]
 use self::rustyline::Editor;
+use std::fs::File;
```

Comment:
I be this gives unused deps when compiling normally right

Reply:
*bet

Reply:
It does, should I add an allod_unused_imports macro?